gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -143,17 +253,24 @@ def get_joint_freq_and_faf(
 
     # Compute group max (popmax) on the merged exomes + genomes frequencies.
     grpmax = pop_max_expr(freq, freq_meta, pops_to_exclude=faf_pops_to_exclude)
-    grpmax = grpmax.annotate(faf95=faf[faf_meta_by_pop.get(grpmax.grp)].faf95)
+    grpmax = grpmax.annotate(faf95=faf[faf_meta_by_pop.get(grpmax.pop)].faf95)
 
     # Annotate Table with all joint exomes + genomes computations.
     ht = ht.annotate(joint_freq=freq, joint_faf=faf, joint_grpmax=grpmax)
+
     ht = ht.annotate_globals(
         joint_freq_meta=freq_meta,
-        joint_freq_index_dict=make_freq_index_dict(freq_meta, label_delimiter="-"),
+        joint_freq_index_dict=make_freq_index_dict_from_meta(
+            freq_meta, label_delimiter="-"
+        ),
         joint_faf_meta=faf_meta,
         joint_faf_index_dict=make_faf_index_dict(faf_meta, label_delimiter="-"),
     )
 
+    # Compute FAF max (fafmax) on the merged exomes + genomes frequencies.
+    fafmax = pop_max_for_faf_expr(ht.joint_faf, ht.joint_faf_meta)
```

Comment:
If were keeping this in, move to L260 and then select it with everything else

Reply:
I had problem to move it to L260 with others, I will test it again once the fafmax PS is merged in gnomad_methods.

Reply:
I found out why, faf_meta from `faf_expr()` is a List, not ArrayExpression, I added hl.literal(faf_meta).